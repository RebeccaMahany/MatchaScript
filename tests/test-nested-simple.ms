function int outer(int firstNum) {
	function int inner(int secondNum) {
		function void evenInner() {
			function void mostInner() {
				print("In mostInner()");
			}	
		}
		return firstNum + secondNum;
	}
	print(inner(3));
	return firstNum;
}
