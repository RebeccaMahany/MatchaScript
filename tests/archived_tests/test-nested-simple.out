In most Inner()
4
1
